src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -192,6 +192,34 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
     return true;
   };
 
+  const validateStartDate = () => {
```

Comment:
Unrelated to this PR.. we have a lot of such date validate code..
We could start thinking about a date validator utility that accepts some rules for validation. Maybe as part of the timezone effort.